feg/gateway/services/n7_n40_proxy/servicers/health.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	"context"
+	"fmt"
+
+	fegprotos "magma/feg/cloud/go/protos"
+	"magma/feg/gateway/services/n7_n40_proxy/metrics"
+	orcprotos "magma/orc8r/lib/go/protos"
+)
+
+// Disable closes all existing pcf connections and disables
+// connection creation for the time specified in the request
+func (srv *CentralSessionController) Disable(ctx context.Context, req *fegprotos.DisableMessage) (*orcprotos.Void, error) {
+	// PCF Connections are stateless HTTP connections, nothing to do here
+	return &orcprotos.Void{}, nil
+}
+
+// Enable enables pcf connection creation
+func (srv *CentralSessionController) Enable(ctx context.Context, void *orcprotos.Void) (*orcprotos.Void, error) {
+	// PCF Connections are stateless HTTP connections, nothing to do here
+	return &orcprotos.Void{}, nil
+}
```

Comment:
Normally we set the grpc end points in the same file so we can see them all in one look. The rest of the functions are in other files (like health)

But it is true that this also make sense this way. 

Let's leave if as it is right now. We can change it in the future if needed. This is another good thing of go, you can move functions around without caring about headers.